src/components/content/Content.js

```diff
@@ -1,11 +1,34 @@
 import './content.scss';
 
+const now = new Date().getHours();
+let text = 'Hello';
+
+if (now >= 22 || now < 3) {
+  text = 'Good night';
+}
+if (now >= 3 && now < 12) {
+  text = 'Good morning';
+}
+if (now >= 12 && now < 18) {
+  text = 'Good afternoon';
+}
+if (now >= 18 && now < 22) {
+  text = 'Good evening';
+}
+
+const Greeting = ({ text, name }) => (
```

Comment:
Component should be as a separate module (otherwise, how I can use it in some other place?).
Date calculating should be also placed inside the component `Greeting`